frontend/simple/views/components/Modal/ModalBasic.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="modal is-active" data-test="modal" v-if="isActive">
+    <div class="modal-background" @click="handleCloseClick"></div>
+    <div class="modal-card" ref="card">
+      <button class="delete" aria-label="close" @click="handleCloseClick"></button>
+      <slot></slot>
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  name: 'ModalBasic',
+  props: {
+    isActive: Boolean
+  },
+  mounted () {
+    global.addEventListener('keyup', this.handleKeyUp)
```

Comment:
Needs a corresponding `beforeDestroy` `removeEventListener`, remember to always clean those up, and better to use Vue.js API to listen for events so that it handles that for you (e.g. have you tried using `v-on:keyup`?)